diktat-rules/src/test/resources/test/paragraph2/kdoc/package/src/main/kotlin/com/saveourtool/diktat/kdoc/methods/KdocWithoutThrowsTagExpected.kt

```diff
@@ -0,0 +1,17 @@
+package test.paragraph2.kdoc.`package`.src.main.kotlin.com.saveourtool.diktat.kdoc.methods
+
+/**
+ * @param onSuccess
+ * @param onFailure
+ */
+fun parseInputNumber(onSuccess: (number: Int) -> Unit, onFailure: () -> Unit) {
+    try {
+        val input: Int = binding.inputEditText.text.toString().toInt()
+        if (input < 0)
+            throw NumberFormatException()
+
+        onSuccess(input)
+    } catch (e: NumberFormatException) {
+        onFailure()
+    }
```

Comment:
I think this functionality is included in the warning tests, and it makes no sense to describe it in the fix test, because in this case there should be no warning, and thus there is no need for a fix.

Reply:
Done